share/picongpu/benchmarks/Thermal/README.rst

```diff
@@ -0,0 +1,11 @@
+Thermal electron benchmark
+==========================
+
+.. sectionauthor:: Rene Widera <r.widera (at) hzdr.de>
+
+This setup simulates a homogeneous electron plasma.
+There are 25 macro-particles per cell which sample a homogeneous distribution of relativistic electrons
+at a density of ρWP = −1020 e/m3.
```

Comment:
This formula seems wrong? I guess you meant `10^20`, but why also `-` os there

Reply:
thx fixed.

Reply:
Yes, 'ρ' typically refers to charge density. The '-' implies that 'e' is the absolute value of the electron's charge. Both is in line with standard physics nomenclature.

Reply:
Okay, but does it hurt to call it `charge density` and not just `density` to avoid any potential confusion